servatrice/src/serversocketinterface.cpp

```diff
@@ -1257,24 +1263,70 @@ Response::ResponseCode AbstractServerSocketInterface::cmdAccountEdit(const Comma
     if (authState != PasswordRight)
         return Response::RespFunctionNotAllowed;
 
-    QString realName = QString::fromStdString(cmd.real_name());
-    QString emailAddress = QString::fromStdString(cmd.email());
-    QString country = QString::fromStdString(cmd.country());
-
+    bool checkedPassword = false;
     QString userName = QString::fromStdString(userInfo->name());
+    if (cmd.has_password_check()) {
+        QString password = QString::fromStdString(cmd.password_check());
+        QString clientId = QString::fromStdString(userInfo->clientid());
```

Comment:
no size checks

Reply:
getting the clientId doesn't need a check, this info comes from the server db